### PR TITLE
refactor: re-enable transaction weights, harmonize runtimes

### PR DESCRIPTION
**Pull request by gregdhill**
Signed-off-by: Gregory Hill <gregorydhill@outlook.com>

### CODE REVIEW

File: parachain/runtime/interlay/src/lib.rs [hunk 1]
```diff
@@ -318,7 +319,10 @@ pub type SlowAdjustingFeeUpdate<R> =
     TargetedFeeAdjustment<R, TargetBlockFullness, AdjustmentVariable, MinimumMultiplier>;
 
 parameter_types! {
-    pub const TransactionByteFee: Balance = 0;  // TODO: this is 0 so that we can do runtime upgrade without fees. Restore value afterwards!
+    // TODO: this is 0 so that we can do runtime upgrade without fees. Restore value afterwards!
+    pub const TransactionByteFee: Balance = 0;
```
Comment (sander2) [inline parachain/runtime/interlay/src/lib.rs:323]: Probably we should set this to something non-zero as well?

Comment (gregdhill) [reply parachain/runtime/interlay/src/lib.rs:323]: We won't want fees initially to do the runtime upgrades as with Kintsugi.

Comment (sander2) [reply parachain/runtime/interlay/src/lib.rs:323]: Right that makes sense

**Comment by sander2** [comment]
One millicent is 1/3_000_000 kint -> parachain upgrade is < megabyte, so fee will be < 1 kint. So should be ok, right?

**Review by sander2 — APPROVED** [review]

**Comment by gregdhill** [comment]
Size of my local wasm blob is 1.8mb (`1.8 * 1024 * 1024 = 1887436`) and `MILLICENTS` is `33333`. So total fee is `33333 * 1887436 = 62913904188` (`~0.06` KINT).

**Pull request merged**